iModelCore/ECDb/ECDb/IssueReporter.h

```diff
@@ -660,6 +660,30 @@ struct ECDB_EXPORT ECDbIssueId
     static ECN::IssueId ECDb_0628;
     static ECN::IssueId ECDb_0629;
     static ECN::IssueId ECDb_0630;
+    
+    static ECN::IssueId ECDb_0631;
```

Comment:
@ColinKerr  @RohitPtnkr1996  I was just going to adjust my other open PR to this when I noticed the IssueIds here are already much hugher than they are on imodel02, and for some reason the IDs are very out of sync. imodel02 is sitting at highest issue id ECDb_0584 while github is on ECDb_0630.
The new IDs that I added for my other PR (585 to 591) on imodel02 are occupied by schemasync on the github stream already.  Now when Rohit merges these new IssueIds down to imodel02, there will be a whole bunch of missing ones in between.

I'm starting to think we should have gone with GUIDs for this. Some dedicated type which is used only for this purpose so we can easily find the code, and then just put a new guid on every issue we add to the code. These sequential numbers maintained in multiple places and being edited without any locking strategy are not working too great.

Reply:
Had a chat with Colin about this, let's keep it as is for now. I'm marking the conversation resolved.